### PR TITLE
Fix capitalization in Plugins settings

### PR DESCRIPTION
**Pull request by Snare-Hawk**
capitalization i noticed

yeah

### CODE REVIEW

**Comment by Iucism** [comment]
cough https://github.com/replugged-org/replugged/commit/42096ae4a5be0fb7bf0a8bbb4a651a2ebf272ef9

**Comment by Snare-Hawk** [comment]
> cough [42096ae](https://github.com/replugged-org/replugged/commit/42096ae4a5be0fb7bf0a8bbb4a651a2ebf272ef9)

yeah
literally like right after i forked, that PR was made and i forgot about it
but there's more than that in this PR

File: i18n/en-US.json [hunk 1]
```diff
@@ -30,20 +30,20 @@
...
   "REPLUGGED_PLUGINS_DISABLE_NOTE": "This will disable the following plugins:",
-  "REPLUGGED_PLUGINS_ENABLE": "Enable plugins",
+  "REPLUGGED_PLUGINS_ENABLE": "Enable Plugins",
   "REPLUGGED_PLUGINS_ENABLE_NOTE": "This will enable the following plugins:",
   "REPLUGGED_PLUGINS_EXPLORE": "Explore Plugin Store",
   "REPLUGGED_PLUGINS_INSTALLED": "Installed Plugins",
   "REPLUGGED_PLUGINS_LICENSE": "License",
-  "REPLUGGED_PLUGINS_LOAD_MISSING": "Load missing plugins",
-  "REPLUGGED_PLUGINS_OPEN_FOLDER": "Open plugins folder",
+  "REPLUGGED_PLUGINS_LOAD_MISSING": "Load missing Plugins",
```
Comment (lleyton) [inline i18n/en-US.json:40]: capitalizing plugins here seems odd

**Review by lleyton — CHANGES_REQUESTED** [review]

File: i18n/en-US.json (hunk 1 above)
Comment (FifiTheBulldog) [reply i18n/en-US.json:40]: I’m ambivalent about capitalizing “plugins” here (Discord seems to use title case for context menu items so maybe it should be capitalized), but either way: the capitalization should be consistent within a string. If “plugins” is capitalized, then “missing” and “folder” should also be capitalized.

Comment (Snare-Hawk) [reply i18n/en-US.json:40]: yeah i forgot to do that here, i thought i had :skull:

**Review by lleyton — APPROVED** [review]

**Pull request merged**